mirna_explorer/repository/create_database.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
Is the shebang necessary?

If you execute the script with python3 create_database.py, then we can remove it since it's not necessary.
However, if you double-click the file or your Mac needs it, then we leave it there.

Personally, I always launch python programs with python3 <file_name>.py so that I don't have to put that line there.

Reply:
It is not necessary, I removed it